src/org/rascalmpl/util/maven/Util.java

```diff
@@ -26,12 +26,41 @@
  */
 package org.rascalmpl.util.maven;
 
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.maven.settings.Settings;
+import org.apache.maven.settings.io.xpp3.SettingsXpp3Reader;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
 
 /* package */ class Util {
```

Comment:
please update with the `feat/native-pom-parsing` branch, as these classes have become public now.